packages/documentation/pages/usage/layouts/navbar.vue

```diff
@@ -43,30 +43,63 @@ notificationData: {
 
 ### Menu
 
-`KtNavbar` only supports this menu structure:
+This is how you display navigation links in `KtNavbar`:
 
 ```js
 menuData: [
 	{
 		title: 'Order Management',
 		links: [
 			{
-				link: '#',
-				title: 'Orders',
 				icon: 'invoice',
 				isActive: false,
+				linkProps: {
+					href: '#',
+				},
+				title: 'Orders',
 			},
 			{
-				link: '#',
-				title: 'Quotes',
 				icon: 'request',
 				isActive: false,
+				linkProps: {
+					href: '#',
+				},
+				title: 'Quotes',
 			},
 		],
 	},
 ]
 ```
 
+The links in KtNavbar are normal `a` elements. However if you want to use one of
+the popular vue router frameworks, you can exchange this for any individual link
+via `v-bind:is`. The data provided in `linkProps` will be bound to the component:
```

Comment:
from an API POV, you don't mention how we bind the component

you need to mention that the link object requires a `component` attribute

so to rephrase this part

```
By default the elements from the `links` prop on `KtNavbar` are bound to an anchor HTML element (`<a />`). Therefore, providing `href` attribute suffices on each element in the array automatically redirects.

In case of the need for a custom component, provide the `component` attribute and any corresponding prop that would need to be bound to it.
```